src/github/middleware.ts

```diff
@@ -31,7 +31,7 @@ const withSentry = function (callback) {
 		try {
 			await callback(context);
 		} catch (err) {
-			logger.error({...err}, "Webhook Error");
+			logger.error(err, "Error while processing webhook");
```

Comment:
Here we need to pass the error itself to have a proper stack grace in the log

Reply:
by just giving it the error, it won't log every property, only stacktrace, message and type I think.  You can do `{...error, err: error}` which should give you best of both worlds.

Reply:
No it doesn't work like that, I tried. You need to pass the exact exception as a first argument otherwise no stack trace